### PR TITLE
docs: readme changes

### PR DESCRIPTION
**Pull request by skirianov**
New README changes according to Team Community remarks and discussion.

## Description

Full redesign and rewrite of the previous README

## Motivation and Context

Previous README was outdated and too generic

## How Has This Been Tested?

N/A

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.

#### Changelog
- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.

#### Tests
- [ ] I have written test cases for my code
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

> I tested manually my code, and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme

#### Code standards
- [ ] My code follows the code style of this project.

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

#### Docs
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,16 @@
 </p>
 
 <h3 align="center">
-  Lightning-Fast Frontend Platform for Headless Commerce
+  <strong>An ecosystem of developer tools that accelerate and simplify building eCommerce Storefronts</strong>
 </h3>
 <p align="center">
-    Vue Storefront is the most popular and most advanced Frontend Platform for eCommerce.
+    Works with any backend
```
Comment (mattmaribojoc) [inline README.md:10]: The team has discussed the wording of "_any_ backend" in the past and have avoided in the past in favor of being more realistic about the primary use case being our pre-built integrations.

File: README.md [hunk 2]
```diff
@@ -121,70 +100,164 @@
...
   </a>
 <br /><br />
-  <a target="_blank" href="https://docs.vuestorefront.io/v2/integrations/">Learn more about available integrations</a>
+  <a target="_blank" href="https://docs.vuestorefront.io/v2/integrations/#ecommerce-platforms">Learn more about available eCommerce platforms</a>
+</p>
+
+## **Getting started** 🚀
+
+### **Step 1: Generate a new project** 🧰
+
```
Comment (mattmaribojoc) [inline README.md:132]: Might be good to call out that this is for the OS integrations, for the enterprise - they have to go through sales

File: README.md [hunk 3]
```diff
@@ -4,10 +4,16 @@
 </p>
 
 <h3 align="center">
-  Lightning-Fast Frontend Platform for Headless Commerce
+  <strong>An ecosystem of developer tools that accelerate and simplify building eCommerce Storefronts</strong>
```
Comment (mattmaribojoc) [inline README.md:7]: There's some inconsistencies in this doc between eCommerce and e-commerce.

**Review by mattmaribojoc — COMMENTED** [review]
Few small comments - overall love this and will use this messaging as a base for some of the core docs

File: README.md (hunk 3 above)
Comment (skirianov) [reply README.md:7]: my bad, it should be eCommerce everywhere

File: README.md [hunk 4]
```diff
@@ -39,78 +45,51 @@
...
+- **Build and Customize**. Build your application with pre-installed [Storefront UI](https://github.com/vuestorefront/storefront-ui) - components library designed for eCommerce, or build your own theme from scratch.
+
+- **Connect with other Integrations**. Vue Storefront is compatible with many third-party services, including StoryBlok, Ampliance, Stripe, Algolia, and more. [See the list of all open-source and enterprise integrations supported](https://docs.vuestorefront.io/v2/integrations/#cms).
+- **Host and Deploy**. Vue Storefront is a server-side rendered application, so you can host it on any server or cloud provider. Enterprise clients can benefit from our in-house [cloud solutions fully optimized for Vue Storefront applications](https://vuestorefront.io/cloud)
+
+<br />
+
+🌟  [Over 1000+ Live Stores, check the list!](https://www.vuestorefront.io/live-projects/?utm_source=github.com&utm_medium=referral&utm_campaign=readme)
+
+## **Supported platforms** 🛍️
```
Comment (filrak) [inline README.md:63]: Lets remove
- Salesforce
- Vendure
- Swell 
- Woo

File: README.md [hunk 5]
```diff
@@ -121,70 +100,164 @@
...
+    <a title="Vendure" target="_blank" href="https://docs.vuestorefront.io/vendure/guide/configuration.html#steps">
+        <img alt="Vendure" width="55" src="https://user-images.githubusercontent.com/1626923/156659153-fafdeb94-a34e-4e9e-8e8a-0f03afde9741.png" />
+    </a>
+<br /><br />
+  <a target="_blank" href="https://docs.vuestorefront.io/v2/integrations/#ecommerce-platforms">Check all available integrations</a>
 </p>
 
 
-## Links
+### **Step 4: Start the project** 🚀
```
Comment (filrak) [inline README.md:198]: I would get rid of step 4 and 5 as they are already part of the per-integration guide

**Review by filrak — CHANGES_REQUESTED** [review]

**Review by mattmaribojoc — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -39,152 +45,211 @@
...
+
+- **Build and Customize**. Build your application with pre-installed [Storefront UI](https://github.com/vuestorefront/storefront-ui) - components library designed for eCommerce, or build your own theme from scratch.
+
+- **Connect with other Integrations**. Vue Storefront is compatible with many third-party services, including StoryBlok, Ampliance, Stripe, Algolia, and more. [See the list of all open-source and enterprise integrations supported](https://docs.vuestorefront.io/v2/integrations/#cms).
+- **Host and Deploy**. Vue Storefront is a server-side rendered application, so you can host it on any server or cloud provider. Enterprise clients can benefit from our in-house [cloud solutions fully optimized for Vue Storefront applications](https://vuestorefront.io/cloud)
+
+<br />
+
+🌟  [Over 2000+ Live Stores, check the list!](https://www.vuestorefront.io/live-projects/?utm_source=github.com&utm_medium=referral&utm_campaign=readme)
```
Comment (filrak) [inline README.md:61]: can we have that at the top, right below the demo link?

**Review by filrak — CHANGES_REQUESTED** [review]

**Review by filrak — APPROVED** [review]

**Pull request merged**